src/main/java/jpo/sdw/depositor/DepositorProperties.java

```diff
@@ -68,20 +70,48 @@ void initialize() {
          subscriptionTopics = DEFAULT_SUBSCRIPTION_TOPICS;
       }
 
-      if (getApiKey() == null) {
+      if (getApiKey() == null || getApiKey().isEmpty()) {
          logger.error("No API Key specified in configuration");
          throw new IllegalArgumentException("No API Key specified in configuration");
       }
 
-      if (getEmailList() == null) {
+      if (getEmailList() == null || getEmailList().isEmpty()) {
          logger.error("No error email list specified in configuration");
          throw new IllegalArgumentException("No error email list specified in configuration");
       }
 
-      if (getEmailFrom() == null) {
+      if (getEmailFrom() == null || getEmailFrom().isEmpty()) {
          logger.error("No from email specified in configuration");
          throw new IllegalArgumentException("No from email specified in configuration");
       }
+
+      if (!emailValid()) {
+         logger.error("From email is not a valid email address");
+         throw new IllegalArgumentException("From email is not a valid email address");
+      }
+
+      if(!emailListValid()){
+         logger.error("Email list is not valid email address(es)");
+         throw new IllegalArgumentException("Email list is not valid email address(es)");
+      }
+   }
+
+   private boolean emailValid() {
+      String emailRegex = "^[a-zA-Z0-9_+&*-]+(?:\\." + "[a-zA-Z0-9_+&*-]+)*@" + "(?:[a-zA-Z0-9-]+\\.)+[a-z"
+            + "A-Z]{2,7}$";
```

Comment:
I noticed that this expression is different than the expression on line 110. I think the expression on 110 could be simplified to just `^[\w+-.%]+@[\w-]+\.[A-Za-z]{2,4}$` to match a single email. Do we want to make these matchers more consistent?

Reply:
The regex on 110 is slightly different and being used to verify a list of emails - we want to be able to send to multiple emails and send from a single one. For consistency, I've updated this regex to be more similar to the list validation (minus the multiple instance capability).